src/js/net/Net.js

```diff
@@ -741,7 +741,7 @@ var Zepto = (function() {
                     names = funcArg(this, name, idx, className(this))
                 names.split(/\s+/g).forEach(function(klass) {
                     (when === undefined ? !$this.hasClass(klass) : when) ?
-                    $this.addClass(klass): $this.removeClass(klass)
```

Comment:
this instance of `removeClass` is not related to DOMUtils, but is instead defined elsewhere in this file, which is adapted from Zepto, a light-weight alternative to jQuery.

We only used Zepto to handle cross-browser variations in making API calls, so this entire `Net.js` class could also probably be replaced with modern JS fetching -- but that's not the issue you're working on, so this change should probably be backed out of the PR, even if this technically works.

Reply:
Ah, I see now that each of your commits has changes to `Net.js` even though that file doesn't reference `DOMUtil.js`.

Please revert all changes to that file.

If you're interested in a follow-up challenge after this one, we could look at simplifying `Net.js` to not include so much never-used code!

Reply:
Hi Joe Germuska! My group and I reverted the changes in the Net.js. Let us know if there are any other changes we should make to our PR. We are interested in the follow-up challenge! Thanks for offering.

Reply:
Since you're doing this for a class, I'm going to be a little bit of a stickler.  First, there is still one outstanding change to `Net.js`; probably the best way to handle it would be to check out the version of that file from the master branch (that is, go back to it) rather than trying to edit it back to being the same.

The other thing is that somehow this PR includes a git submodule named `TimelineJS3`, which shows up as a directory within the repository. That shouldn't be in here. 

Can you try to remove those things? In a pinch, I could probably fix the PR before merging it, but if you're trying to learn to use git, I don't want to just change it without you knowing.

Reply:
Hi again Joe! I am sorry for the wait and thank you for your patience. I was able to successfully revert the changes done to Net.js and delete the submodule. 

I really did appreciate you, giving us an opportunity to better understand and learn git!